data/sql/updates/pending_db_world/rev_1599321019527522400.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1599321019527522400');
+
+UPDATE `creature_template` SET `modelid1` = 30583 WHERE `entry` = 32315;
```

Comment:
```suggestion
UPDATE `creature` SET `id` = 32315, `modelid` = 30583 WHERE `guid` = 200984;
```

Reply:
@Kitzunu 
``` sql
UPDATE `creature` SET `id` = 32315, `modelid` = 30583, `curhealth`='4183500' WHERE `guid` = 200984;
```
Do it like this in case some guys were going back and forth with fixing the issue temporarily.